lib/ruby_smb/smb1/packet.rb

```diff
@@ -0,0 +1,8 @@
+require 'bit-struct'
```

Comment:
Move this require into `lib/ruby_smb.rb`

While you're at it, do the same with the requires in `client.rb`

Let's make sure that all requires are in that central location other than stuff from stdlib.